gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStats.java

```diff
@@ -13,168 +13,113 @@
  */
 package io.trino.gateway.ha.clustermonitor;
 
-import java.util.Map;
-import java.util.Objects;
+import com.google.common.collect.ImmutableMap;
 
-import static com.google.common.base.MoreObjects.toStringHelper;
+import java.util.Map;
 
-public class ClusterStats
+import static java.util.Objects.requireNonNull;
+
+public record ClusterStats(
+        String clusterId, int runningQueryCount,
+        int queuedQueryCount,
+        int blockedQueryCount,
```

Comment:
`blockedQueryCount` is not used anywhere. It should correspond to `BasicQueryStats.isFullyBlocked()`, as mentioned in https://github.com/trinodb/trino-gateway/issues/233. 

I think we should remove it, as it doesn't seem like an important dimension of cluster health. What is your opinion?

Reply:
Good catch. Let's remove the field in a follow-up.

Reply:
https://github.com/trinodb/trino-gateway/pull/241